sdk/digitaltwins/Azure.DigitalTwins.Core/tests/QueryTests.cs

```diff
@@ -97,13 +99,35 @@ public async Task Query_PaginationWorks()
                 // act
                 var options = new QueryOptions();
                 options.MaxItemsPerPage = pageSize;
-                AsyncPageable<string> asyncPageableResponse = client.QueryAsync(queryString, options);
+
+                var queryWaitTimeoutStopwatch = new Stopwatch();
```

Comment:
fyi you can do `Stopwatch.StartNew()` to make this and only take 1 statement instead of 2. :)